src/main/groovy/com/palantir/gradle/gitversion/GitVersionPlugin.groovy

```diff
@@ -88,8 +94,23 @@ class GitVersionPlugin implements Plugin<Project> {
             // first to preserve this behavior in cases where this call would fail but native "git" call does not.
             new DescribeCommand(git.getRepository()).call()
 
-            return GitCli.runGitCommand(project.rootDir, "describe", "--tags", "--always", "--first-parent",
-                    "--match=${prefix}*")
+            /*
+             * Mimick 'git describe --tags --always --first-parent --match=${prefix}*' by using rev-list to
+             * support versions of git < 1.8.4
+             */
+            int depth = 0
+            String revList = GitCli.runGitCommand(project.rootDir, "rev-list", "--first-parent", "HEAD")
```

Comment:
Tiny concern that this might be a tiny bit slower by loading up all commits on develop, but I tried this on a 2 year old internal repo and it was only 2000ish hashes, so probably nbd.

If it turns out to be a problem I'm sure we can page our way through the rev-list